NIPostsReload/Presentation/Scenes/NIPostFeedModule/ViewState/NIPostViewState.swift

```diff
@@ -12,7 +12,7 @@ struct NIPostViewState {
         let postId: Int
         let title: String
         let previewText: String
-        let date: String
+        let date: Date
```

Comment:
It would be more effective to store the posts and the view state separately. This way, whenever the posts are updated, a new view state can be generated, leading to an update in the view.

Reply:
The viewState should not be stored as a property. Instead, it should be created within the updateView method.